desktop/app/lib/keychain/index.js

```diff
@@ -1,26 +1,31 @@
-const keytar = require( 'keytar' );
+const { safeStorage } = require( 'electron' );
+const ElectronStore = require( 'electron-store' );
 
 const keychainService = 'WordPress.com';
+const store = new ElectronStore( {
+	name: keychainService,
+	watch: true,
+} );
 
 async function write( key, value ) {
-	return keytar.setPassword( keychainService, key, JSON.stringify( value ) );
+	if ( ! safeStorage.isEncryptionAvailable() ) {
+		throw new Error( 'Encryption is not avaialble.' );
+	}
+
+	const buffer = safeStorage.encryptString( value );
+	store.set( key, buffer.toString( 'latin1' ) );
```

Comment:
Why latin1, and not utf-8?

Reply:
Good question! I took it from the source I referenced, which uses `latin1`. Probably a good idea to use `utf8` as you suggest, though.